### PR TITLE
adding the link to each step for issue 2747

### LINKED ISSUE #2747: Virtual Intern tutorial steps. The arrow at the bottom of the page that points to the next step could be a link that navigates you to the next step
### Problem
When I was doing the step by step instructions, I would get to the bottom of the page and expect to be able to click on a link or button to the next step but I had to go back up and find the next step. 

### Steps to reproduce the problem

Go to each step after step one and scroll to bottom 

### Screenshots

Screen Shot 2019-10-02 at 10 32 00 PM

### Proposed solution

Either make a HTML arrow/image that can take you to the next step or make the "Step #" be a link to the next step.

### PR DESCRIPTION
**Pull request by garret123h**
Fixes #2747 

### Description
I added the links to the next steps for virtual interns. I got rid of the link to step 8 because they just need to return to the first steps for step 8.

### Raw.Githack preview link

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-planet-installation-vagrant.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-configurations-vagrant.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-vagrant.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-docker-tutorial.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-github-and-markdown.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-planetapps.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-github-and-repositories.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-github-issues.md

https://raw.githack.com/garret123h/garret123h.github.io/link-to-steps/#!pages/vi/vi-nation.md

### CODE REVIEW

**Comment by perryc85** [comment]
@garret123h - I think you forgot step 6. You should also fix your preview links.

**Comment by garret123h** [comment]
Ok forgot to update the branch and added the step 6 link in

**Comment by garret123h** [comment]
Ok I changed the links to relative if you want to look at it again.

File: pages/vi/vi-github-issues.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ This is an exercise to help you familiarize with GitHub issues, committing, and
 [Helpful links and videos](vi-faq.md#Helpful_Links)
 [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
 
-## Next Section _(Step 7)_ **→**
+## Next Section _([Step 7](vi-github-and-repositories.md))_ **→**
```
Comment (paulbert) [inline pages/vi/vi-github-issues.md]: @garret123h Looks like this one goes back to Step 5 rather than on to Step 7

**Pull request merged**